runtime/compiler/env/VMJ9.cpp

```diff
@@ -2460,7 +2460,7 @@ TR_J9VMBase::getClassSignature_DEPRECATED(TR_OpaqueClassBlock * clazz, int32_t &
    for (i = 0; i < numDims; i++)
       sig[i] = '[';
    if (* name != '[')
-      sig[i++] = 'L';
+      sig[i++] = 'L';  //do we need to worry about Q here?
```

Comment:
I think this will need to check whether `myclass` is an inline/value type class.  I'm in the process of adding a method - `isValueType(TR_OpaqueClass*)` - to this class to check that.